integration_test/oath_test.dart

```diff
@@ -89,36 +299,36 @@ void main() {
   });
 
   group('Password tests', () {
-    /// note that the password groups should be run as whole
-
-    /// TODO implement test for password replacement
-    /// appTest('OATH: replace oath password', (WidgetTester tester) async {
-    ///    await tester.replaceOathPassword('aaa111', 'bbb222');
-    /// });
-
-    // cannot restart the app on Android to be able to unlock
+    // NOTE: that the password groups should be run as whole
+    // NOTE: cannot restart the app on Android to be able to unlock: skip
     group('Desktop password tests', skip: isAndroid, () {
-      var testPassword = 'testPassword';
-
-      appTest('Set OATH password', (WidgetTester tester) async {
-        await tester.setOathPassword(testPassword);
+      var firstPassword = 'firstPassword';
+      var secondPassword = 'secondPassword';
+      var thirdPassword = 'thirdPassword';
+      appTest('Set first OATH password', (WidgetTester tester) async {
+        // await tester.resetOath();
```

Comment:
should the comment be removed so that the oath session is reset? Otherwise remove the commented line.